test/Rx/Functional/React/PromiseToObservableTest.php

```diff
@@ -40,7 +40,9 @@ function () {
      */
     public function from_promise_failure()
     {
-        $p = Promise::rejected(new Exception('error'));
+        $p = new ReactPromise(function () {
+            1 / 0;
```

Comment:
nit: it'd be nice to add a comment here to explain we're trying to trigger an `Error`. :) I'll merge as is though.

Reply:
@asm89 I would have added that comment .... :D